### PR TITLE
Fix a grammatical error in the python example.

### PR DESCRIPTION
**Pull request by hong9lol**
## Changes
I found a grammatical error in the python example. please check this out.

### CODE REVIEW

File: specification/metrics/api.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ consider:
 # Python
 
 exception_counter.Add(1, {"exception_type": "IOError", "handled_by_user": True})
-exception_counter.Add(1, exception_type="IOError", handled_by_user=True})
+exception_counter.Add(1, exception_type="IOError", handled_by_user=True)
```
Comment (carlosalberto) [inline specification/metrics/api.md:349]: Based on the previous example, I guess a `{` is missing instead? cc @ocelotl

Comment (ocelotl) [reply specification/metrics/api.md:349]: Right, @carlosalberto. I would also change `Add` to `add`, it is more usual in Python :+1:

**Comment by tigrannajaryan** [comment]
@hong9lol please sign the CLA otherwise it is impossible to merge the PR.

**Review by reyang — APPROVED** [review]

File: specification/metrics/api.md (hunk 1 above)
Comment (reyang) [reply specification/metrics/api.md:349]: The 1st one is showing that users can pass in a dict.
The 2nd one is showing that users can pass in named parameters (`**params`), so `}` should be removed.



> I would also change `Add` to `add`, it is more usual in Python 👍

+1

**Review by carlosalberto — APPROVED** [review]

**Comment by carlosalberto** [comment]
@hong9lol Please rename the method to `add` since you are editing this, please :D (once that is done, we will merge it)

**Review by MrAlias — APPROVED** [review]

**Comment by hong9lol** [comment]
> @hong9lol Please rename the method to `add` since you are editing this, please :D (once that is done, we will merge it)

Oh, yes I will

**Review by arminru — APPROVED** [review]

**Pull request merged**